one-page.js

```diff
@@ -373,6 +365,36 @@ function inlineAssets(projectPath) {
                 }
             })();
 
+            // Add Snapchat/Google CTA code if needed
+            (function() {
+                switch(config.one_page.cta_option) {
+                case 'snapchat':
+                    console.log("↪️ Adding Snapchat Ad CTA code");
+                    addLibraryFile('snapchat-cta.js');
+                break;
+                case 'google':
+                    console.log("↪️ Adding Google Ad CTA code");
+                    var allowedValues = ["portrait", "landscape", "portrait,landscape"];
+                    if(config.one_page.adOrientation && allowedValues.includes(config.one_page.adOrientation)) {
+                        var orientation = config.one_page.adOrientation;
+                        var size = (orientation == 'portrait') ? 'width=320,height=480' : 'width=480,height=320\n';
+                        var googlePrimaryAssets = `    <meta name="ad.orientation" content="${orientation}">\n    <meta name="ad.size" content="${size}">`;
+                        var exitApi = `<script type="text/javascript" src="https://tpc.googlesyndication.com/pagead/gadgets/html5/api/exitapi.js"></script>\n`;
```

Comment:
Instead of adding it explicitly like this, it would be good to have a function like `addLibraryFile` instead so this logic is reusable for future networks